models/platform-limits/src/index.ts

```diff
@@ -0,0 +1,7 @@
+// Export types
+export * from './business-units/types';
+export * from './limit-with-current/types';
+
+// Export models
+export * as BusinessUnitLimitsProjection from './business-units';
```

Comment:
should i change the folder name also reflect the API model?
`business-unit-limits-projection` ? or is it OK to keep it same as in FE test-data?

Reply:
I vote for keeping it like it currently is as the nested folder nature already tell us this is a platform limit